packages/react-use-intercom/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # react-use-intercom
 
+## 5.4.2
+
+### Patch Changes
+
+- d5c7433: Add support for Content-Security-Policy nonce
+
```

Comment:
You can skip all the `CHANGELOG` changes and version bump. Run the [changeset script](https://github.com/devrnt/react-use-intercom/blob/main/package.json#L18) and follow the prompt, changeset will take care of the rest

Reply:
I wanted to however didn't wanne go through downgrading pnpm to install the packages

Reply:
No need to install pnpm, you can run `npx @changesets/cli@2.26.0`

Reply:
Done 👍